src/lib/bcmp/bm/bm_util.c

```diff
@@ -1,4 +1,4 @@
 #include "bm_util.h"
 
-const ip6_addr_t multicast_global_addr = {{0x3FF,0x0,0x0,0x1000000}, 0};
-const ip6_addr_t multicast_ll_addr = {{0x2FF,0x0,0x0,0x1000000}, 0};
+// const ip6_addr_t multicast_global_addr = {{0x3FF,0x0,0x0,0x1000000}, 0};
+// const ip6_addr_t multicast_ll_addr = {{0x2FF,0x0,0x0,0x1000000}, 0};
```

Comment:
now defined in `bm_common/util.h`, which has been updated with the latest bm_core update.